src/pages/index.js

```diff
@@ -3,25 +3,40 @@ import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import { configuration } from "../utils/constants.js";
 import Section from "../components/Section.js";
+import Popup from "../components/Popup.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from "../components/UserInfo.js";
 import { initialCards } from "../utils/constants.js";
 import { userConfigData } from "../utils/constants.js";
 import "./index.css"
 import Api from "../components/Api.js";
+import { _ } from "core-js";
 
 // select elements
 const editButton = document.querySelector('.profile__edit-button');
-const addCardButton = document.querySelector('.profile__add-button')
+const addCardButton = document.querySelector('.profile__add-button');
 const profileFormElement = document.forms['profileForm'];
 const cardAddFormElement = document.forms['cardForm'];
+const avatarUpdateFormElement = document.forms['updateAvatar']
 const nameInput = profileFormElement.querySelector('.modal__name');
 const jobInput = profileFormElement.querySelector('.modal__description');
+const avatar = document.querySelector('.profile__photo');
+const avatarOverlay = document.querySelector('.profile__photo-overlay')
 
 // create classes
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "32400e4e-bdb6-4554-9f2b-339b0c55f4e6",
+    "Content-Type": "application/json",
+  }
+});
+const avatarValidator = new FormValidator(configuration, avatarUpdateFormElement)
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store the validator using the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```